cli/tools/test.rs

```diff
@@ -306,7 +286,12 @@ async fn test_specifier(
     None
   };
 
-  worker.execute_main_module(&test_specifier).await?;
+  // We only execute the specifier as a module if it is tagged with TestMode::Module or
+  // TestMode::Both.
```

Comment:
```suggestion
  // We only execute the specifier as a main module if it is tagged with TestMode::Module or
  // TestMode::Both.
```

Reply:
Side module actually.

Reply:
Aha, okay, didn't undestand the comment then.